### ISSUE TITLE
Remove Direct2D stuff from master

### ISSUE THREAD

**Issue by pyscripter** [SynEdit/SynEdit#93]
I would vote to take all Direct2D stuff out of master and place it in a branch until it is production ready.   There is no point in further "polluting" the already full of IFDEFs code with experimental stuff.  

I think the same should be done for the Kylix stuff.   Who uses Kylix today anyway?

I get the go ahead I could submit a PR implementing this.

**Comment by pyscripter** [comment]
I gave up working with this branch.  I maintain a fork of https://github.com/TurboPack-Tokyo/SynEdit at https://github.com/pyscripter/SynEdit-2.  Check it out.

**Comment by Memnarch** [comment]
intsead of a bazillion forks, maybe ask for access to this repo?

**Comment by pyscripter** [comment]
I did try.  A fast and stable implementation of code-folding was the first major advancement in Synedit in the last 10 years.  I did my best to contribute it and adapt it this repository and Turbopower Synedit.

Synedit is a very old project and the result of contributions by some of the brightest pascal programmers.   However it lacked good project management and it is left behind compared to other modern editors such as VS code, scintilla and pascal based ATSynedit( https://github.com/Alexey-T/ATSynEdit).

The problem with this repository, is that it aims to maintain compatibility with very old Delphi versions.  It is extremely hard to change/improve anything for the fear of breaking compatibility with some ancient Delphi version.  Half the code is IFDEFs.   More recently this was compounded with half-baked stuff entering the master branch.

TurboPack Synedit went a long way in modernising the code and getting rid of the IFDEF nightmare.  However TurboPack Synedit is no longer maintained.  Hence my fork.
